components/conversation.js

```diff
@@ -203,8 +207,16 @@ function Conversation(assistant, config) {
     // if audio tries to come in when we are sending text, bail out
     if (sendingText) return;
 
-    const request = AssistRequest.create({ audioIn: data });
-    conversation.write(request);
+    // Take the audio data and split it up into bite-sized chunks that google can digest
+    // and then send it along. Under ordinary circumstances, we should be capturing 
+    // audio data fast enough to send in a single request so that the assistant can be 
+    // highly responsive. 
+    let chunks = [];
+    while(chunks.length < data.length / MAX_FRAME_LENGTH) {
+      chunks.push(data.slice(chunks.length * MAX_FRAME_LENGTH, Math.min(data.length, (chunks.length + 1) * MAX_FRAME_LENGTH)));
```

Comment:
(I've never done reviewing in github, so forgive my noobness)

Can you break your Math.min into a const and give it a name explaining what it's doing?